<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DodajProdukt</class>
 <widget class="QDialog" name="DodajProdukt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>761</width>
    <height>914</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>MS Sans Serif</family>
    <pointsize>10</pointsize>
    <italic>true</italic>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_6">
   <item row="0" column="0">
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>741</width>
        <height>894</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox">
         <property name="font">
          <font>
           <pointsize>18</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="title">
          <string>DODAJANJE PRODUKTA</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0">
           <layout class="QVBoxLayout" name="verticalLayout_9">
            <item>
             <widget class="QGroupBox" name="groupBox_2">
              <property name="font">
               <font>
                <pointsize>12</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="title">
               <string>Podatki produkta</string>
              </property>
              <layout class="QGridLayout" name="gridLayout">
               <item row="0" column="0">
                <layout class="QVBoxLayout" name="verticalLayout_5">
                 <item>
                  <layout class="QVBoxLayout" name="verticalLayout">
                   <item>
                    <widget class="QLabel" name="label_2">
                     <property name="font">
                      <font>
                       <family>MS Sans Serif</family>
                       <pointsize>10</pointsize>
                       <weight>50</weight>
                       <italic>true</italic>
                       <bold>false</bold>
                      </font>
                     </property>
                     <property name="frameShape">
                      <enum>QFrame::WinPanel</enum>
                     </property>
                     <property name="frameShadow">
                      <enum>QFrame::Raised</enum>
                     </property>
                     <property name="lineWidth">
                      <number>1</number>
                     </property>
                     <property name="midLineWidth">
                      <number>0</number>
                     </property>
                     <property name="text">
                      <string>Stranka</string>
                     </property>
                     <property name="buddy">
                      <cstring>comboBox_podjetje</cstring>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QComboBox" name="comboBox_podjetje">
                     <property name="font">
                      <font>
                       <family>MS Sans Serif</family>
                       <pointsize>10</pointsize>
                       <weight>50</weight>
                       <italic>true</italic>
                       <bold>false</bold>
                      </font>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <layout class="QVBoxLayout" name="verticalLayout_2">
                   <item>
                    <widget class="QLabel" name="label_5">
                     <property name="font">
                      <font>
                       <family>MS Sans Serif</family>
                       <pointsize>10</pointsize>
                       <weight>50</weight>
                       <italic>true</italic>
                       <bold>false</bold>
                      </font>
                     </property>
                     <property name="frameShape">
                      <enum>QFrame::WinPanel</enum>
                     </property>
                     <property name="frameShadow">
                      <enum>QFrame::Raised</enum>
                     </property>
                     <property name="lineWidth">
                      <number>1</number>
                     </property>
                     <property name="midLineWidth">
                      <number>0</number>
                     </property>
                     <property name="text">
                      <string>ID produkta</string>
                     </property>
                     <property name="buddy">
                      <cstring>lineEdit_id</cstring>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLineEdit" name="lineEdit_id">
                     <property name="font">
                      <font>
                       <family>MS Sans Serif</family>
                       <pointsize>10</pointsize>
                       <weight>50</weight>
                       <italic>true</italic>
                       <bold>false</bold>
                      </font>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <layout class="QVBoxLayout" name="verticalLayout_3">
                   <item>
                    <widget class="QLabel" name="label_3">
                     <property name="font">
                      <font>
                       <family>MS Sans Serif</family>
                       <pointsize>10</pointsize>
                       <weight>50</weight>
                       <italic>true</italic>
                       <bold>false</bold>
                      </font>
                     </property>
                     <property name="frameShape">
                      <enum>QFrame::WinPanel</enum>
                     </property>
                     <property name="frameShadow">
                      <enum>QFrame::Raised</enum>
                     </property>
                     <property name="lineWidth">
                      <number>1</number>
                     </property>
                     <property name="midLineWidth">
                      <number>0</number>
                     </property>
                     <property name="text">
                      <string>Naziv produkta</string>
                     </property>
                     <property name="buddy">
                      <cstring>lineEdit_nazivProdukta</cstring>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLineEdit" name="lineEdit_nazivProdukta">
                     <property name="font">
                      <font>
                       <family>MS Sans Serif</family>
                       <pointsize>10</pointsize>
                       <weight>50</weight>
                       <italic>true</italic>
                       <bold>false</bold>
                      </font>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <layout class="QVBoxLayout" name="verticalLayout_4">
                   <item>
                    <widget class="QLabel" name="label_4">
                     <property name="font">
                      <font>
                       <family>MS Sans Serif</family>
                       <pointsize>10</pointsize>
                       <weight>50</weight>
                       <italic>true</italic>
                       <bold>false</bold>
                      </font>
                     </property>
                     <property name="frameShape">
                      <enum>QFrame::WinPanel</enum>
                     </property>
                     <property name="frameShadow">
                      <enum>QFrame::Raised</enum>
                     </property>
                     <property name="lineWidth">
                      <number>1</number>
                     </property>
                     <property name="midLineWidth">
                      <number>0</number>
                     </property>
                     <property name="text">
                      <string>Cena produkta</string>
                     </property>
                     <property name="buddy">
                      <cstring>lineEdit_cena</cstring>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLineEdit" name="lineEdit_cena">
                     <property name="font">
                      <font>
                       <family>MS Sans Serif</family>
                       <pointsize>10</pointsize>
                       <weight>50</weight>
                       <italic>true</italic>
                       <bold>false</bold>
                      </font>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_6">
              <item>
               <widget class="Line" name="line_5">
                <property name="font">
                 <font>
                  <family>MS Sans Serif</family>
                  <italic>true</italic>
                 </font>
                </property>
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout">
                <item>
                 <widget class="QPushButton" name="pushButton_popravi">
                  <property name="font">
                   <font>
                    <family>MS Sans Serif</family>
                    <pointsize>12</pointsize>
                    <italic>true</italic>
                   </font>
                  </property>
                  <property name="text">
                   <string>Popravi podatke</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="pushButton_dodaj">
                  <property name="font">
                   <font>
                    <family>MS Sans Serif</family>
                    <pointsize>12</pointsize>
                    <italic>true</italic>
                   </font>
                  </property>
                  <property name="text">
                   <string>Dodaj produkt</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <widget class="Line" name="line_3">
                <property name="font">
                 <font>
                  <family>MS Sans Serif</family>
                  <italic>true</italic>
                 </font>
                </property>
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_8">
              <item>
               <widget class="QGroupBox" name="groupBox_3">
                <property name="font">
                 <font>
                  <pointsize>12</pointsize>
                  <weight>75</weight>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="title">
                 <string>Iskalnik produktov</string>
                </property>
                <layout class="QGridLayout" name="gridLayout_2">
                 <item row="0" column="0">
                  <layout class="QVBoxLayout" name="verticalLayout_7">
                   <item>
                    <widget class="QPushButton" name="pushButton_isci">
                     <property name="enabled">
                      <bool>true</bool>
                     </property>
                     <property name="font">
                      <font>
                       <pointsize>12</pointsize>
                      </font>
                     </property>
                     <property name="text">
                      <string>Isci</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLineEdit" name="lineEdit_isci">
                     <property name="font">
                      <font>
                       <family>MS Sans Serif</family>
                       <pointsize>10</pointsize>
                       <weight>50</weight>
                       <italic>true</italic>
                       <bold>false</bold>
                      </font>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="Line" name="line_4">
                <property name="font">
                 <font>
                  <family>MS Sans Serif</family>
                  <italic>true</italic>
                 </font>
                </property>
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QGroupBox" name="groupBox_4">
              <property name="font">
               <font>
                <pointsize>12</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="title">
               <string>Produkti</string>
              </property>
              <layout class="QGridLayout" name="gridLayout_3">
               <item row="0" column="0">
                <widget class="QTreeWidget" name="treeWidget">
                 <property name="font">
                  <font>
                   <family>MS Sans Serif</family>
                   <pointsize>10</pointsize>
                   <italic>true</italic>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">QHeaderView::section{padding:7px 15px}</string>
                 </property>
                 <column>
                  <property name="text">
                   <string>ID</string>
                  </property>
                  <property name="font">
                   <font>
                    <family>MS Sans Serif</family>
                    <pointsize>10</pointsize>
                    <italic>true</italic>
                   </font>
                  </property>
                  <property name="textAlignment">
                   <set>AlignCenter</set>
                  </property>
                 </column>
                 <column>
                  <property name="text">
                   <string>Naziv</string>
                  </property>
                  <property name="font">
                   <font>
                    <family>MS Sans Serif</family>
                    <pointsize>10</pointsize>
                    <italic>true</italic>
                   </font>
                  </property>
                  <property name="textAlignment">
                   <set>AlignCenter</set>
                  </property>
                 </column>
                 <column>
                  <property name="text">
                   <string>Cena €</string>
                  </property>
                  <property name="font">
                   <font>
                    <family>MS Sans Serif</family>
                    <pointsize>10</pointsize>
                    <italic>true</italic>
                   </font>
                  </property>
                  <property name="textAlignment">
                   <set>AlignCenter</set>
                  </property>
                 </column>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="Line" name="line_2">
              <property name="font">
               <font>
                <family>MS Sans Serif</family>
                <italic>true</italic>
               </font>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton">
              <property name="font">
               <font>
                <family>MS Sans Serif</family>
                <pointsize>12</pointsize>
                <italic>true</italic>
               </font>
              </property>
              <property name="text">
               <string>Izhod</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>pushButton_isci</tabstop>
  <tabstop>comboBox_podjetje</tabstop>
  <tabstop>lineEdit_id</tabstop>
  <tabstop>lineEdit_nazivProdukta</tabstop>
  <tabstop>lineEdit_cena</tabstop>
  <tabstop>pushButton_popravi</tabstop>
  <tabstop>pushButton_dodaj</tabstop>
  <tabstop>lineEdit_isci</tabstop>
  <tabstop>treeWidget</tabstop>
  <tabstop>pushButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>DodajProdukt</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>689</x>
     <y>661</y>
    </hint>
    <hint type="destinationlabel">
     <x>294</x>
     <y>169</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
